web/styles/left_sidebar.css

```diff
@@ -313,6 +313,73 @@
     }
 }
 
+.stream_search_section {
+    position: sticky;
+    top: 0;
+    /* Must be more than .stream-list-subsection-header */
+    z-index: 3;
+    /* Must be padding not margin so that the sticky headers don't show behind it */
+    padding: var(--left-sidebar-sections-vertical-gutter)
+        var(--left-sidebar-right-margin) 3px 5px;
+}
+
+.stream-list-subsection-header {
+    display: grid;
+    align-items: center;
+    grid-template:
+        "arrow row-content scroll-buffer" var(
+            --line-height-sidebar-row-prominent
+        )
+        / var(--right-sidebar-header-icon-toggle-width) minmax(0, 1fr);
+    cursor: pointer;
+    background-color: var(--color-background);
+    position: sticky;
+    /* Input is 29px tall at 16px/1em; plus 2px for top/bottom border.
+       Input container has 8px and 3px top/bottom padding = 11px total padding */
+    top: calc(1.8125em + 2px + 11px);
```

Comment:
I corrected this value this afternoon; left a note on the commit for @evykassirer's sake, but that can be removed once they've seen it.

Reply:
Thanks!